### PR TITLE
Updating checkout command to prompt version and related entities

### PR DESCRIPTION
**Pull request by lucasncabral**
```
$ ml-git models checkout model-ex --wizard
Define the dataset version to be downloaded [latest]: 5
Do you want to checkout related dataset [y/N]?
Do you want to checkout related labels [y/N]?
```

### CODE REVIEW

File: ml_git/commands/utils.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 from click import UsageError
 
+
```
Comment (Vieira-Carlos) [inline ml_git/commands/utils.py:9]: Typo?

**Review by Vieira-Carlos — APPROVED** [review]

File: ml_git/commands/utils.py (hunk 1 above)
Comment (lucasncabral) [reply ml_git/commands/utils.py]: It was a typo, thanks. The blank line was removed at [05bc3de](https://github.com/HPInc/ml-git/pull/237/commits/05bc3de9d9793bdceaf630fc6cf73885f1d9e4a5).

File: docs/mlgit_commands.md [hunk 2]
```diff
@@ -130,6 +130,8 @@ Options:
   --fail-limit INTEGER  Number of failures before aborting the command
                         [default: no limit].
   --full                Show all contents for each directory.
+  --wizard              Enable the wizard to prompt the user requesting
+                        information when needed.
```
Comment (andersonpablosilva) [inline docs/mlgit_commands.md:134]: Just a suggestion, maybe we can update this help message:

```suggestion
  --wizard              Enable the wizard to request information when needed.
```

If you accept this suggestion, remember to update it in the other commands.

**Review by MatheusClaud — APPROVED** [review]

**Review by MatheusClaudi — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]

File: docs/mlgit_commands.md (hunk 2 above)
Comment (lucasncabral) [reply docs/mlgit_commands.md]: The suggestion was applied in 9a4f421. All commands have been updated

**Pull request merged**